projects/pipelines/testing/config.yaml

```diff
@@ -125,9 +141,9 @@ ldap:
 rhods:
   catalog:
     image: brew.registry.redhat.io/rh-osbs/iib
-    tag: 655658
-    channel: stable
-    version: 2.6.0
+    tag: 682686
+    channel: fast
```

Comment:
not important, but no need to change the channel, the RCs are published in these channels:
> fast,alpha,embedded, stable, stable-2.8, eus-2.8